avocado/plugins/yaml_to_mux.py

```diff
@@ -45,6 +46,23 @@
 __RE_FILE_SUBS = re.compile(r'(?<!\\)\\:')  # substitute '\\:' but not '\\\\:'
 
 
+class _BaseLoader(Loader):
+    """
+    YAML loader with additional features related to mux
+    """
+    pass
+
+
+_BaseLoader.add_constructor(u'!include', lambda loader,
```

Comment:
Because by modifying the original `Loader` I modify the `yaml.Loader` which causes the issue reported by Vincent. Therefor I only modify the `_BaseLoader`. Then later in code we have to create yet another instance, because the the loader accepts `node_cls` and each loader should use different constructor.